### PR TITLE
Fix ts error for any type

### PR DESCRIPTION
**Pull request by HalinaPP**

### CODE REVIEW

File: src/utils/date.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+  long = 'long',
+  numeric = 'numeric',
+  '2-digit' = '2-digit',
+  short = 'short',
+  narrow = 'narrow',
+}
+
+export const monthsForLocale = (
+  localeName = 'en-EN',
+  monthFormat = DateMonthFromat.long,
```
Comment (akrutsko) [inline src/utils/date.ts:11]:
```suggestion
  monthFormat: Intl.DateTimeFormatOptions = { month: 'long' },
```

File: src/utils/date.ts [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+) => {
   const now = new Date();
   const format = new Intl.DateTimeFormat(localeName, { month: monthFormat })
```
Comment (akrutsko) [inline src/utils/date.ts:14]:
```suggestion
  const { format } = new Intl.DateTimeFormat(localeName, monthFormat);
```

File: src/utils/date.ts [hunk 3]
```diff
@@ -1,4 +1,15 @@
-export const monthsForLocale = (localeName = 'en-EN', monthFormat = 'long') => {
+export enum DateMonthFromat {
```
Comment (akrutsko) [inline src/utils/date.ts:1]: Intl.DateTimeFormatOptions built-in

File: src/utils/date.ts (hunk 1 above)
Comment (HalinaPP) [reply src/utils/date.ts]: Thank you, fixed

File: src/utils/date.ts (hunk 3 above)
Comment (HalinaPP) [reply src/utils/date.ts]: Thank you, fixed

**Pull request merged**